src/bundle/Resources/views/dashboard/all_content_tab.html.twig

```diff
@@ -0,0 +1,34 @@
+{% if data|length %}
+    <table class="ez-table-data">
+        <thead>
+        <tr>
+            <th>{{ 'dashboard.table.name'|trans }}</th>
```

Comment:
We use the `desc` function from jms translation bundle to define the default translation, so that it gets dumped by the translation extractor (see other templates from the same package).